clvm_tools/curry.py

```diff
@@ -13,7 +13,7 @@
 
 CURRY_OBJ_CODE = assemble(
     """
-(a (q (c 4 (c 2 (c 5 (c 7 0))))) (c (q (c (q . 2) (c (c (q . 1) 5) (c (a 6 (c 2 (c 11 (q 1)))) 0))) (c (i 5 (q 4 (q . 4) (c (c (q . 1) 9) (c (a 6 (c 2 (c 13 (c 11 0)))) 0))) (q . 11)) 1)) 1))
+(a (q #a 4 (c 2 (c 5 (c 7 0)))) (c (q (c (q . 2) (c (c (q . 1) 5) (c (a 6 (c 2 (c 11 (q 1)))) 0))) #a (i 5 (q 4 (q . 4) (c (c (q . 1) 9) (c (a 6 (c 2 (c 13 (c 11 0)))) 0))) (q . 11)) 1) 1))
```

Comment:
How is `#a` interpreted here?

Reply:
It means "the atom representing the `a` opcode", so it does a keyword look-up from `clvm`.